apps/captiveportal/captiveportal.pro

```diff
@@ -0,0 +1,54 @@
+QT += qml quick gui gui-private dbus sql
+# The name of your app
+TARGET = sailfish-captiveportal
+
+CONFIG += link_pkgconfig
+
+TARGETPATH = /usr/bin
+target.path = $$TARGETPATH
+
+INSTALLS += target
+PKGCONFIG +=  mlite5 sailfishwebengine
+INCLUDEPATH += $$system(pkg-config --cflags sailfishwebengine)
+
+packagesExist(qdeclarative5-boostable) {
+    message("Building with qdeclarative-boostable support")
+    DEFINES += HAS_BOOSTER
+    PKGCONFIG += qdeclarative5-boostable
+} else {
+    warning("qdeclarative5-boostable not available; startup times will be slower")
+}
+
+# Translations
+TS_PATH = $$PWD
+TS_FILE = $$OUT_PWD/sailfish-browser.ts
```

Comment:
A question: is it a problem that both captiveportal and browser define the same translation files? Is there a conflict?

Reply:
I had in my notes that this should be split as well but I'd not like to disturb translations now with this. Need to think about.

I think this can be solved by moving the translation.pri stuff to the apps directory so that we run it through only once.